### PR TITLE
feat(saved-search): Add components to save current search

### PR DESCRIPTION
**Pull request by lynnagara**
Ref: SEN-396

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
is `OrganizationSavedSearchSelector` behind a feature flag?

**Comment by lynnagara** [comment]
@billyvg It is - https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/stream/filters.jsx#L74-L75

File: src/sentry/static/sentry/app/views/stream/filters.jsx [hunk 1]
```diff
@@ -89,8 +89,10 @@ class StreamFilters extends React.Component {
               )}
             >
               <OrganizationSavedSearchSelector
+                orgId={orgId}
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/stream/filters.jsx]: I modified this component to take the entire organization in #12522

File: src/sentry/static/sentry/app/views/stream/organizationSavedSearchSelector.jsx [hunk 2]
```diff
@@ -64,12 +69,109 @@ export default class OrganizationSavedSearchSelector extends React.Component {
           }
         >
           {this.renderList()}
+          <StyledMenuItem divider={true} />
+          <ButtonBar>
+            <SaveSearchButton query={query} orgId={orgId} />
+          </ButtonBar>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/stream/organizationSavedSearchSelector.jsx]: In the spec only manager and owner roles are allowed to create saved searches.

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/stream/organizationSavedSearchSelector.jsx]: I imagine this should include admin as well? I'll add a check for `project:write`

**Review by markstory — APPROVED** [review]
I am assuming that actually saving searches will come separately?

**Comment by lynnagara** [comment]
Yes, will follow up. This design is also going to change quite a bit but I'll merge this first.

**Pull request merged**